From d46e105418ebde9e64324f9039e56f113291d80e Mon Sep 17 00:00:00 2001
From: Pavan S <psreenivas@habana.ai>
Date: Wed, 15 Mar 2023 15:45:26 +0200
Subject: [PATCH] [SW-136144] gaudi3: remove SPECS_H9_DIR env

The SPECS_DIR is overridden tfor gaudi3 to SPECS_H9_ROOT
Hence removing SPECS_H9_DIR variable. However the zephyr
applications for gaudi3 need to include SPECS_ROOT for
including pci_ids header file hence keeping SPECS_ROOT
and SPECS_DIR in zephyr Makefile.

Signed-off-by: Pavan S <psreenivas@habana.ai>
Change-Id: I6ce45df5d62d7f89e309000b74b073e84b48ad56
---

diff --git a/package/zephyr/zephyr.mk b/package/zephyr/zephyr.mk
index d1e511c..a892ce6 100644
--- a/package/zephyr/zephyr.mk
+++ b/package/zephyr/zephyr.mk
@@ -14,7 +14,7 @@
 ZEPHYR_CONF_OPTS += -DEXTRA_CFLAGS=-Werror -Wno-misleading-indentation
 ZEPHYR_CONF_OPTS += -DBUILD_VERSION="$(shell cut -d- -f2- $(ZEPHYR_BUILDDIR)/localversion-habanalabs)"
 ZEPHYR_CONF_OPTS += -DBUILD_MACHINE="$(shell hostname -s)"
-ZEPHYR_CONF_OPTS += -DCMAKE_C_FLAGS="-I$(SPECS_EMBEDDED_DIR) -I$(SPECS_EMBEDDED_DIR)/$(DEV_PREFIX) -I$(SPECS_ROOT) -I$(SPECS_EXT_ROOT) -I$(SPECS_ROOT)/$(DEV_PREFIX)/asic_reg -I $(SPECS_H9_DIR) -I$(SPECS_ROOT_H9)"
+ZEPHYR_CONF_OPTS += -DCMAKE_C_FLAGS="-I$(SPECS_EMBEDDED_DIR) -I$(SPECS_EMBEDDED_DIR)/$(DEV_PREFIX) -I$(SPECS_ROOT) -I$(SPECS_EXT_ROOT) -I$(SPECS_DIR)/$(DEV_PREFIX)/asic_reg -I $(SPECS_DIR)"
 ZEPHYR_CONF_OPTS += -DDTS_EXTRA_CPPFLAGS="-I$(SPECS_EMBEDDED_DIR)"
 
 DEVICE_NAME = $(firstword $(subst _, ,$(HL_DEV)))
diff --git a/scripts/make/gaudi3/Makefile b/scripts/make/gaudi3/Makefile
index 9354f3f..ee5c634 100644
--- a/scripts/make/gaudi3/Makefile
+++ b/scripts/make/gaudi3/Makefile
@@ -1,4 +1,4 @@
-export SPECS_H9_DIR ?= $(SPECS_ROOT_H9)/gaudi3/asic_reg
+export SPECS_DIR = $(SPECS_ROOT_H9)
 
 .PHONY: gaudi3-post-image
 gaudi3-post-image:
